app/javascript/packs/responseTemplates.js

```diff
@@ -0,0 +1,35 @@
+import loadingImg from '../../assets/images/loading-ellipsis.svg'
+import { handleLoggedOut, fetchResponseTemplates, addReplyObservers, addToggleListener } from '../responseTemplates/responseTemplates';
+
+const userStatus = document.querySelector('body').getAttribute('data-user-status');
+
+function prepareButton() {
+  const button = document.querySelector('.response-templates-button')
+  if (!button) {
+    return
+  }
+
+  button.addEventListener('click', () => {
+    const responsesWrapper = document.querySelector('.mod-responses-container');
+    responsesWrapper.style.display = 'flex';
```

Comment:
Can this `display: flex` be added to a CSS class instead? Less JS and in a CSS class it gets cached.